news/_posts/2024-06-11-GalaxyTrainingAcademy-Call-Contribution.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: news
+title: "Open Call for Trainers for the Galaxy Training Academy"
+tags:
+  - gtn
+contributions:
+  authorship:
+    - teresa-m
+---
+
+# Open Call for Trainers for the Galaxy Training Academy
+
+We are happy to announce the Galaxy Training Academy 2024, a global online training event, where learners can get to know Galaxy and learn how to analyze data for various research topics.
+
+
+**Dates**: October 07-11, 2024
+**Location**: Global online
+**Registration deadline**: July the 12th
+
+## The event
+
+The event will provide a self-paced online learning experience for learners who would like to dive into the depths of Galaxy data analysis. The event is similar to the Smörgåsbord, but the program will be more structured. On Monday learners can get to know Galaxy and its tremendous functionality. Between Tuesday and Thursday, learners can choose from different Tracks that they are interested in. This can help learn to either get onboarded or understand fassates of one research area more in-depth. It is still up to the learnes to choose there program for the week but the track will give them guidance on what order will help to achieve their individual learning goals. In there self-paced learning endevers they will be supported via chat by us.
+
+### Initial Program:
+
+
+<table>
+  <tr>
+   <td><strong>Day</strong></td>
+   <td><strong>Program</strong></td>
+ </tr>
+ <tr>
+   <td>Monday</td>
+   <td>Galaxy Introduction</td>
+ </tr>
+ <tr>
+   <td>Tuesday-
+     <p> Thurday
+   </td>
+   <td>Different tracks:
+   <ul>
+        <li>Proteomics</li>
+        <li>Assembly</li>
+        <li>Transcriptomics</li>
+        <li>Microbiome</li>
+        <li>Machine learning</li>
+        <li>BY-COVID</li>
+      </ul>
+   </td>
+ </tr>
+ <tr>
+  <td>Friday</td>
+  <td>Choose from a variety of different topics</td>
+ </tr>
+</table> 
```

Comment:
```suggestion
Day | Program
--- | ---
Monday | Galaxy Introduction
Tuesday-Thurday | Different Tracks: Proteomics, Assembly, Transcriptomics, Microbiome, Machine Learning, BY-COVID 
Friday |Choose from a variety of different topics
```

maybe it's a bit simpler. (Also I did not know you all were doing by-covid as a track! This is pleasant news, @wm75 are you supporting that? Is it just the one-health bits or the RO-crates as well?)

Reply:
I thougt it would be nice to have all tracks in one line. But I guess that is no needed.